git/foundations_git/git_basics.md

```diff
@@ -26,105 +26,78 @@ This section contains a general overview of topics that you will learn in this l
 
 1. <span id="new-github-repo"></span>You should have already created a GitHub account in the [Setting Up Git](https://www.theodinproject.com/lessons/foundations-setting-up-git) lesson.
 
-1. Create a new repository by clicking the button shown in the screenshot below.
-
-    ![The GitHub Profile Screen](https://cdn.statically.io/gh/TheOdinProject/curriculum/b54d14c5dcee1c6fac61aee02fca7e9ef7ba1510/foundations/git_basics/project_practicing_git_basics/imgs/00.png)
+1. From the GitHub homepage, create a new repository by clicking the "+" button in the top-right corner and selecting "New repository".
 
 1. Give your repository the name "git_test" in the repository name input field. Check "Add a README file". And then create the repository by clicking the "Create repository" button at the bottom of the page.
 
-    ![Create new repo using GitHub](https://cdn.statically.io/gh/TheOdinProject/curriculum/d0579120dd641d26aaef6a601008e998f8a7c648/git/foundations_git/git_basics/imgs/01.png)
-
-1. This will redirect you to your new repository on GitHub. To get ready to copy (clone) this repository onto your local machine, click the green "Code" button. Then select the SSH option, and copy the line below it. **NOTE: You MUST click the SSH option to get the correct URL.**
-
-    ![Copy SSH link using GitHub](https://cdn.statically.io/gh/TheOdinProject/curriculum/b54d14c5dcee1c6fac61aee02fca7e9ef7ba1510/foundations/git_basics/project_practicing_git_basics/imgs/02.png)
+1. This will redirect you to your new repository on GitHub. You should see a layout that includes a tab list of different views (such as "Code", "Pull Requests", and so on) and your repository name along the top of the page, along with a lot of other content elsewhere on the page. To get ready to copy (clone) this repository onto your local machine, click the "Code" button which should be in the section below your repository name. Then select the SSH option in the "Clone" section, and copy the line below it. **NOTE: You MUST click the SSH option to get the correct URL.**
```

Comment:
```suggestion
1. This will redirect you to your new repository on GitHub. You should see a layout that includes a tab list of different views (such as "Code", "Pull Requests", and so on) and your repository name along the top of the page, along with a lot of other content elsewhere on the page. To get ready to copy (clone) this repository onto your local machine, click the "Code" button which should be in the section **below** your repository name - note that you should not click the "Code" *tab* that appears above your repository name. Then select the SSH option in the "Clone" section, and copy the line below it. **NOTE: You MUST click the SSH option to get the correct URL.**
```
Lemme know what you think of the above addition. Realized it may be confusing that there's a "Code" tab and button on the same page.

Reply:
I would delete this part "You should see a layout that includes a tab list of different views (such as "Code", "Pull Requests", and so on) and your repository name along the top of the page, along with a lot of other content elsewhere on the page." to not focus attention on that other Code button. 

And instead of "click the "Code" button which should be in the section **below** your repository name - note that you should not click the "Code" *tab* that appears above your repository name." I would say "Click the green "Code" button at the bottom right of the repository name."

Reply:
So you're thinking something more along the lines of:

> 1. This will redirect you to your new repository on GitHub. To get ready to copy (clone) this repository onto your local machine, click the green "Code" button which should be below and to the right of your repository name. Then select the SSH option in the "Clone" section, and copy the line below it. **NOTE: You MUST click the SSH option to get the correct URL.**

That would probably be good. An alternative would be instead of mentioning its position based on the repo name, using the branch toggle or the repo files/folders, e.g. "click the green "Code" button, which should be to the right of the button displaying the current branch (typically it will display the `main` branch)" or "click the green "Code" button, which should be directly above the section that contains the files and folders in your directory"

I'll leave it up to you what makes the most sense from a user perspective, though.

Reply:
> So you're thinking something more along the lines of:
> 
> > 1. This will redirect you to your new repository on GitHub. To get ready to copy (clone) this repository onto your local machine, click the green "Code" button which should be below and to the right of your repository name. Then select the SSH option in the "Clone" section, and copy the line below it. **NOTE: You MUST click the SSH option to get the correct URL.**
> 
> That would probably be good. An alternative would be instead of mentioning its position based on the repo name, using the branch toggle or the repo files/folders, e.g. "click the green "Code" button, which should be to the right of the button displaying the current branch (typically it will display the `main` branch)" or "click the green "Code" button, which should be directly above the section that contains the files and folders in your directory"
> 
> I'll leave it up to you what makes the most sense from a user perspective, though.

I'm gonna go with relating the "Code" button position to branch name, as that doesn't change on smaller viewports